Paintroid/src/main/java/org/catrobat/paintroid/dialog/PermissionInfoDialog.java

```diff
@@ -66,16 +66,16 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 	@Override
 	public Dialog onCreateDialog(Bundle savedInstanceState) {
 		return new AlertDialog.Builder(getContext(), R.style.PocketPaintAlertDialog)
-				.setIcon(permissionType.getIconResource())
-				.setMessage(permissionType.getMessageResource())
-				.setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
-					@Override
-					public void onClick(DialogInterface dialog, int which) {
-						ActivityCompat.requestPermissions(getActivity(), permissions, requestCode);
-					}
-				})
-				.setNegativeButton(android.R.string.cancel, null)
-				.create();
+					.setIcon(permissionType.getIconResource())
+					.setMessage(permissionType.getMessageResource())
+					.setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
+						@Override
+						public void onClick(DialogInterface dialog, int which) {
+							ActivityCompat.requestPermissions(getActivity(), permissions, requestCode);
+						}
+					})
+					.setNegativeButton(android.R.string.cancel, null)
+					.create();
```

Comment:
PLEASE CLARIFY: Any special reason for these additional tabs?

Reply:
somehow these tabs dont exist in the commit i uploaded, no idea where they are from
and even the old version on github has one tab more than in my local commit

Reply:
just saw that these seem to be 8 spaces long, while my system has a tab at 4 spaces

Reply:
@SebastianGrief or @ThomasSchwengler , do you have an idea of what could go on here?

Reply:
![image](https://user-images.githubusercontent.com/38420601/78215693-66757680-74b8-11ea-8bc0-1a130939a6ea.png)
(1) github
![image](https://user-images.githubusercontent.com/38420601/78215820-bbb18800-74b8-11ea-9d8a-1b94c85454c4.png)
(2) local Paintroid-120
![image](https://user-images.githubusercontent.com/38420601/78216231-a5f09280-74b9-11ea-9f82-4510c1394cb4.png)
(3) old version 

I've installed a chrome extension to see whitespaces on github. [1]

For some reason the tabs aren't the same size.(1) The count is the same as local though. (2)
Are you sure they weren't added by you (maybe added automatically for some reason)? @geolas94 They are missing on the branch I'm currently working on. (3) 


[1] https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom/related

Reply:
I was asking because this file was otherwise not changed, and if we get more such changes in the future, it would unnecessarily increase the number of lines we have to look at during code reviews.

Reply:
True. I missed that it wasn't changed at all sorry.

Reviews for whitespace only changes can be turned off. But thats only temporary help.

We should maybe have a look at the next few PRs to confirm that it wasn't just a Android Studio problem for this commit.

Reply:
So, should I ignore it and merge the PR as it is? But why not have @geolas94 push a change that reverts this first, as it has no apparent value and just increases the size of the PR...

Reply:
It should be reverted. Android studio has an vcs feature and I'm sure this change shows up there.

Reply:
It should be reverted yes.

Reply:
This change is visible in gitk:
![image](https://user-images.githubusercontent.com/30626927/78223041-87dd5f00-74c6-11ea-91cd-2047f457e518.png)

And Android Studio:
![image](https://user-images.githubusercontent.com/30626927/78223183-c115cf00-74c6-11ea-939c-5b4db26447e2.png)

Reply:
You can revert it graphically in Android Studio:

![screen](https://user-images.githubusercontent.com/30626927/78223602-79437780-74c7-11ea-9303-f4d524f42d74.png)

While at it, please also revert the changes to the german translations, they don't belong into this pull request.

Reply:
Right --- I missed this about the German translations, wasn't looking too closely, sorry.